rundeckapp/grails-spa/packages/ui-trellis/theme/scss/custom/exec-status.scss

```diff
@@ -59,9 +59,9 @@
 
   &.aborted,
   &[data-execstate=ABORTED] {
-    color: var(--dark-gray);
+    color: var(--danger-color);
```

Comment:
normally aborted is shown as gray, i think maybe in dark mode it should be light-gray and light mode should be dark-gray. I don't think it should be "danger"